### PR TITLE
In order refund page disable buttons if all order is refunded

### PR DESCRIPTION
**Pull request by Tomanhez**
Screenshot 2019-10-15 at 10 07 23

### CODE REVIEW

File: src/Resources/views/orderRefunds.html.twig [hunk 1]
```diff
@@ -6,6 +6,12 @@
 
 {% set customer = order.customer %}
 
+{% set disableButton = '' %}
+
+{% if order.paymentState == 'refunded' %}
+    {% set disableButton = 'disabled' %}
+{% endif %}
```
Comment (pamil) [inline src/Resources/views/orderRefunds.html.twig]: Can be refactored into:

```twig
{% set disableButton = order.paymentState == 'refunded' ? 'disabled' : '' %}
```

File: src/Resources/views/orderRefunds.html.twig [hunk 2]
```diff
@@ -6,6 +6,10 @@
 
 {% set customer = order.customer %}
 
+{% set disableButton = '' %}
```
Comment (pamil) [inline src/Resources/views/orderRefunds.html.twig]: This should be not needed anymore.

File: tests/Behat/Context/Ui/RefundingContext.php [hunk 3]
```diff
@@ -283,6 +283,16 @@ public function shouldStillBeAbleToRefundOrderShipment(): void
         Assert::true($this->orderRefundsPage->isOrderShipmentAvailableToRefund());
     }
 
+    /**
+     * @Then I should not be able to refund anything
+     */
+    public function iShouldNotBeAbleToRefundAnything(): void
+    {
+        Assert::false($this->orderRefundsPage->canRefund());
```
Comment (lchrusciel) [inline tests/Behat/Context/Ui/RefundingContext.php]: Do we need all three methods here?

File: src/Resources/views/orderRefunds.html.twig [hunk 4]
```diff
@@ -6,6 +6,8 @@
 
 {% set customer = order.customer %}
```
Comment (GSadee) [inline src/Resources/views/orderRefunds.html.twig]: Unnecessary blank line

File: src/Resources/views/orderRefunds.html.twig [hunk 5]
```diff
@@ -6,6 +6,8 @@
...
 
+{% set disableButton = order.paymentState == 'refunded' ? 'disabled' : '' %}
```
Comment (GSadee) [inline src/Resources/views/orderRefunds.html.twig]: Use constant state instead of string `refunded`

**Comment by GSadee** [comment]
Thank you, Tomasz! :1st_place_medal:

**Pull request merged**